lib/omnibus/packagers/bff.rb

```diff
@@ -200,6 +203,24 @@ def create_bff_file_name
       "#{safe_base_package_name}-#{project.build_version}-#{project.build_iteration}.#{safe_architecture}.bff"
     end
 
+    #
+    # Remove files with colons in their name.
+    #
+    # This is to satisfy the post-install file check done by the "sysck" utility
+    # called by installp. Without this, expect messages similar to the below
+    # (in reference to filename "/opt/harmony/embedded/share/man/man3/Tie::Memoize.3":
+    # sysck: 3001-022 The file .../share/man/man3/Tie was not found."
+    #
+    # @return [void]
+    #
+    def remove_colon_filenames
+      log.info(log_key) { "Removing colon filenames" }
+      files = Dir.glob("#{staging_dir}/opt/**/*").select {|file| file.match(/.*::.*/)}
+      files.each do |file|
+        File.delete(file)
+        log.debug(log_key) { "Removed `#{file}'" }
```

Comment:
In general, you want to log _before_ doing an operation in omnibus land because of how the log messages are output. I would move this up. If the file.delete operation fails, you won't have the preceeding log message to hint at. I would also include why it's being removed:

```ruby
log.debug(log_key) { "Removing `#{file}' because it contained colons" }
File.delete(file)
```